NAS-102329: Update docs for default view in Accounts/Users (#1278)
freenas/freenas-docs: userguide/accounts.rst

   Managing User Accounts

> By default, each user entry displays the username, User ID (UID), if the

Reviewer: ```suggestion
By default, each user entry displays the username, User ID (UID), whether the
```

> user is built in to %brand%, and the full name of the user. This table

Reviewer: ```suggestion
user is built into %brand%, and full name. This table
```
